plugins/Delay/DelayControlsDialog.cpp

```diff
@@ -22,10 +22,13 @@
  *
  */
 
-#include "delaycontrolsdialog.h"
-#include "delaycontrols.h"
+#include "DelayControlsDialog.h"
+#include "DelayControls.h"
 #include "embed.h"
 #include "TempoSyncKnob.h"
+#include "../Eq/EqFader.h"
```

Comment:
Is this functionality that all faders should eventually leverage, or is it specific to the way the plugins use them?

If so, we may choose to call the old Faders `OldFader` and slowly phase them out in favor of the new one.

If they can be combined (relatively easily), we may choose to add a new constructor which acts as a flag to use old vs. new.

If they are better left separate, it is more a matter of nomanclature.... I'm not sure I like the prefix "SelfUpdating" although I can't think of a better one just yet.